### PR TITLE
stacks: don't show landing as blocked if Phab API token is present (Bug 1801926)

### PR DESCRIPTION
**Pull request by cgsheeh**

### CODE REVIEW

File: landoui/templates/stack/stack.html
Comment (zzzeid) [inline landoui/templates/stack/stack.html:107]: I think we may be overloading this button a little too much.

A few options:

- We could rename the button to "Landing Options" (instead of "Preview Landing")
- The we could separate the landing section from the uplift section in a way that one or the other could be hidden depending on the state of the stack
- At this point, we could keep the landing options button enabled if either of the two things are possible


Alternatively, and perhaps more cleanly, we could move the "request uplift" functionality outside of the landing modal altogether. If it's just a dropdown and button, what is the value of showing the landing preview anyway?

This check (as to whether something is landable or not) is going to get a lot more rigorous with the revision worker changes, and I'm hesitant to overload the functionality of this button.

**Review by zzzeid — CHANGES_REQUESTED** [review]

File: landoui/templates/stack/stack.html
Comment (cgsheeh) [reply landoui/templates/stack/stack.html:107]: After discussion with glob, we would like to try and land this change to unblock the web UI pilot. I'm going to work on moving this button outside the landing modal and onto the base stack page, tracked with https://bugzilla.mozilla.org/show_bug.cgi?id=1801959.

Comment (cgsheeh) [reply landoui/templates/stack/stack.html:107]: Actually, hold off on that plan for a moment.

**Comment by cgsheeh** [comment]
This can be reviewed now, and will be landed/deployed with the other uplift related patches I've been sending. This is a quick fix while I implement https://bugzilla.mozilla.org/show_bug.cgi?id=1801959

**Review by zzzeid — APPROVED** [review]

**Pull request merged**
